Description: Remove plugin_as_library since it was removed from lint
 error: lint `plugin_as_library` has been removed: plugins have been deprecated and retired
 This permits the docs test to pass.
Last-Update: 2023-02-27
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/src/lib.rs
+++ b/src/lib.rs
@@ -33,7 +33,6 @@
     missing_docs,
     non_shorthand_field_patterns,
     overflowing_literals,
-    plugin_as_library,
     stable_features,
     unconditional_recursion,
     unknown_lints,
